internal/connector/internal/api/public/model_processor.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * Connector Management API
+ *
+ * Connector Management API is a REST API to manage connectors.
+ *
+ * API version: 0.1.0
+ * Contact: rhosak-support@redhat.com
+ * Generated by: OpenAPI Generator (https://openapi-generator.tech)
+ */
+
+package public
+
+import (
+	"time"
+)
+
+// Processor struct for Processor
+type Processor struct {
```

Comment:
I assume a `Processor` will live on a `connector_cluster`.. so I have not added anything specific for `Processor`'s.